[simulate_descriptor.py]
  [test_simulate_descriptor]
    expected: ERROR
